### PR TITLE
[tests-only][full-ci] add test to check add and remove member acivities

### PR DESCRIPTION
**Pull request by prashant-gurung899**
## Description
This PR adds test where a sharer checks api activities of a space after a member is added and removed.
``` feature
Scenario: check add member to space activity
Scenario: check remove member from space activity
```

## Related Issue

- Related to: https://github.com/owncloud/ocis/issues/9543

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiActivities/shareActivities.feature [hunk 1]
```diff
@@ -497,3 +497,173 @@ Feature: check share activity
         }
       }
       """
+
+
+  Scenario: sharer checks (add/remove) member activity
```
Comment (saw-jan) [inline tests/acceptance/features/apiActivities/shareActivities.feature:502]: or something
```suggestion
  Scenario: check add member to sapce activity
```

File: tests/acceptance/features/apiActivities/shareActivities.feature [hunk 2]
```diff
@@ -497,3 +497,173 @@ Feature: check share activity
...
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has sent the following space share invitation:
+      | space           | new-space    |
+      | sharee          | Brian        |
+      | shareType       | user         |
+      | permissionsRole | Space Editor |
+    And user "Alice" has removed the access of user "Brian" from space "new-space"
+    When user "Alice" lists the activities of space "new-space" using the Graph API
```
Comment (saw-jan) [inline tests/acceptance/features/apiActivities/shareActivities.feature:511]: we can have two separate scenarios for addition and removal
```suggestion
      | permissionsRole | Space Editor |
    When user "Alice" lists the activities of space "new-space" using the Graph API
```

Comment (prashant-gurung899) [reply tests/acceptance/features/apiActivities/shareActivities.feature:511]: Sure, I will separate them into two scenarios.

**Review by saw-jan — COMMENTED** [review]
LGTM :+1: 
Please, rebase and resolve the conflicts

**Review by saw-jan — APPROVED** [review]

**Pull request merged**